src/import/FSHImporter.ts

```diff
@@ -514,6 +521,18 @@ export class FSHImporter extends FSHVisitor {
     return rules;
   }
 
+  visitCaretValueRule(ctx: pc.CaretValueRuleContext): CaretValueRule {
+    const path = ctx.path() ? this.visitPath(ctx.path()) : null;
```

Comment:
Do you think it's weird to have a `null` path?  I'm wondering if it would be better to use empty string `''` to signify the path at the root?

Reply:
Keeping the path a string if we can seems like a good reason to do `''`, and I can't think of any good reason to make it `null`, so yeah empty string works for me.